src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import duke.exception.WriteToStorageException;
+import duke.task.Task;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskListTest {
+
+    @Test
+    void getStore() {
```

Comment:
Any reason why this method getStore is left blank? Noticed a couple of methods below left blank as well.